docs/fastdds/xml_configuration/common.rst

```diff
@@ -82,6 +83,43 @@ The following example shows the implementation of one locator of each transport
     :start-after: <!-->XML-LOCATOR-LIST<-->
     :end-before: <!--><-->
 
+.. _externalLocatorListType:
+
+ExternalLocatorListType
+^^^^^^^^^^^^^^^^^^^^^^^
+
+It represents a list of external locator entries.
+Each entry can be a ``<udpv4>`` or a ``<udpv6>`` tag.
+These tags can be configured with the following attributes:
+
++---------------------+--------------------------------------------------------+--------------------+------------------+
+| Name                | Description                                            | Values             | Default          |
++=====================+========================================================+====================+==================+
+| ``externality``     | Number of hops from the participant's host to the      | ``uint8_t``        | 1                |
+|                     | LAN represented by the external locator.               |                    |                  |
+|                     | Valid values: from 1 to 255.                           |                    |                  |
++---------------------+--------------------------------------------------------+--------------------+------------------+
+| ``cost``            | Communication cost relative to other locators on the   | ``uint8_t``        | 0                |
+|                     | same externality level.                                |                    |                  |
+|                     | Valid values: from 0 to 255.                           |                    |                  |
++---------------------+--------------------------------------------------------+--------------------+------------------+
+| ``mask``            | Number of significant bits on the LAN represented by   | ``uint8_t``        | 24               |
+|                     | the external locator.                                  |                    |                  |
+|                     | Valid values: from 1 to 31 (UDPv4) or 127 (UDPv6)      |                    |                  |
++---------------------+--------------------------------------------------------+--------------------+------------------+
```

Comment:
I suggest adding the `|br|` macro so the table is formatted with line breaks and can be viewed with the webpage margins correctly. Right now, the last two columns are not viewed at first.

Reply:
fcb8c80